app/presenters/data_set_csv_presenter.rb

```diff
@@ -17,11 +17,48 @@ def to_csv
 
   def to_array_for_csv
     [].tap do |csv|
-      headers = ['name', 'address1', 'address2', 'town', 'postcode', 'access_notes', 'general_notes', 'url', 'lat', 'lng', 'phone', 'fax', 'text_phone']
-      csv << headers
+      csv << all_headers
       places.each do |place|
-        csv << headers.collect { |h| place.send(h.to_sym) }
+        csv << build_row(place)
       end
     end
   end
+
+private
+
+  # We need to handle the location fields separately from the others.
+  # location is a Point object (so doesn't serialize well to CSV by default)
+  # and doesn't need to be exported because it's either overridden from the
+  # override fields or set from MapIt's data using the postcode.
+  # Also exclude the id from the CSV since it isn't needed by the import.
+  def non_location_headers
+    Place.attribute_names - ['_id', 'location', 'override_lng', 'override_lat']
+  end
```

Comment:
We could memoize this method as we're going to call it multiple times per csv file.  I assume that Mongoid will memoize `attribute_names` already (fingers crossed), but it can't hurt to avoid the extra `Array#-` (which in turn will create a new array).